tests/h/views/api_auth_test.py

```diff
@@ -79,7 +79,10 @@ def test_post_returns_state(self, post_auth_ctrl, pyramid_request):
         assert ctx['state'] == 'a_random_string'
 
     @pytest.fixture
-    def auth_ctrl(self, factories, pyramid_config, pyramid_request):
+    def auth_ctrl(self, pyramid_config, pyramid_request):
+        """
+        Configure a valid request for `OAuthAuthorizeController.get`.
```

Comment:
This method actually returns a new `OAuthAuthorizeController` configured with a valid `GET` request, doesn't it? A `"Return a new ..."` docstring might be clearer.